libs/scripts/scripts.js

```diff
@@ -123,7 +123,8 @@ const config = {
   geoRouting: 'on',
   fallbackRouting: 'on',
   links: 'on',
-  imsClientId: 'milo',
+  // will be reverted before merging
+  imsClientId: 'fedsmilo',
```

Comment:
Did we have any query param to set this? Or did I just see this in a branch of rares

Reply:
That was in a branch of Rares. The query param thing needed to be removed anyway before merging, so I just changed the id. On Rares' branch it made sense because we were not merging into main.